ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/proxy/cco-manual-workload-identity/filestore-csi/deprovision/cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity-filestore-csi-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-proxy-cco-manual-workload-identity-filestore-csi-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-gcp-ipi-proxy-deprovision
+  - ref: ipi-conf-gcp-filestore-workload-identity-federation-cleanup
```

Comment:
In the step, I saw:
```
if test -f "${SHARED_DIR}/proxy-conf.sh"
then
        # shellcheck disable=SC1091
        source "${SHARED_DIR}/proxy-conf.sh"
        logger "INFO" "Loaded proxy configuration from ${SHARED_DIR}/proxy-conf.sh"
fi
```

The proxy is used to access cluster api, while there is no access to cluster, so I do not think we need the block.